### DISCUSSION TITLE
Items with same skins, can we fix this?

### DISCUSSION THREAD

**Discussion by TehNo0b** [Crowns-of-Power/CoP#13] [Ideas]
Alot of our items use the same skins.  CP, DT are good examples.  In CP no matter what robe you wear they are all just black, why?  I'd be willing to fix all of these if no one else has the time if I thought the new skins would be used.  Nothing crazy.. just basic color changes.  Doesn't cost us anything performance wise, or use up anymore datablocks it's just a small image file we have to download one time.  No item should look the same.

**Comment by gothrowhell2** [comment]
We could prob do pallet swaps, but i would be down for this, sounds easy enough.
